src/pages/__tests__/createStore.test.ts

```diff
@@ -43,7 +44,8 @@ const mockChangedStoredState: SavedState = {
   playgroundSourceChapter: Constants.defaultSourceChapter,
   playgroundSourceVariant: Variant.DEFAULT,
   playgroundExternalLibrary: 'NONE' as ExternalLibraryName,
-  playgroundLang: SupportedLanguage.JAVASCRIPT
+  playgroundLang: SupportedLanguage.JAVASCRIPT,
+  playgroundLanguage: defaultLanguageConfig
```

Comment:
Can these fields be better named?

Reply:
`playgroundLang` is meant to be deprecated and removed once `playgroundLanguage` (which contains the entire config, not just the `mainLanguage`, which is what `playgroundLang` is) set up &rarr; see task 3 of #2463.

`playgroundLang` was added recently in #2444, in a rush, as we wanted to quickly add the language dropdown selector so that people can start experimenting with the non-Source languages. Its value is equal to `playgroundLanguage.mainLanguage`, so this will be made redundant.

I'm keeping it in this PR as the components that depend on `playgroundLang` are not ready yet for the migration.